doc-build/md055.md

```diff
@@ -0,0 +1,20 @@
+This rule is triggered when a table is missing a border. Borders consist of the
```

Comment:
Similar comment as above, maybe something like "... triggered when a table is missing a leading or trailing pipe character."